### PR TITLE
pins not ignoring packages

### PR DESCRIPTION
**Pull request by toddbeckman-wf**
# Description

Actually ignore pin infractions for ignored packages.

# Testing

- [ ] CI passes
- [ ] Pick your favorite CI process last release broke and try it with this branch.

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on HipChat: InfoSec Forum._

File: lib/dependency_validator.dart [hunk 1]
```diff
@@ -263,17 +263,21 @@ void run({
 ///
 /// package: ^1.2.3
 /// package: ">=1.2.3 <2.0.0"
-void checkPubpspecYamlForPins(YamlMap pubspecYaml, {bool fatal: true}) {
+void checkPubpspecYamlForPins(
```
Comment (travissanderson-wf) [inline lib/dependency_validator.dart]: nit: this method has a typo in the name "Pubpspec" vs "Pubspec"

**Comment by travissanderson-wf** [comment]
QA +1 works on locally for DT to ignore dart_style pin

**Comment by jayudey-wf** [comment]
+10
  - installed into a repo w/ a failing CI and it's passing as expected now

**Comment by jayudey-wf** [comment]
QA +1

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
  - see above
- [ ] Security review (if required)
- [ ] Unit tests created/updated
- [ ] All unit tests pass
- [ ] Rosie has run and reports properly the release the ticket will be included in

Merging into master.

**Comment by jayudey-wf** [comment]
@Workiva/release-management-pp

**Pull request merged**